core/local/local.go

```diff
@@ -336,7 +336,7 @@ func (e *ExecutionScheduler) runExecutor(
 func (e *ExecutionScheduler) Run(globalCtx, runCtx context.Context, engineOut chan<- stats.SampleContainer) error {
 	executorsCount := len(e.executors)
 	logger := e.logger.WithField("phase", "local-execution-scheduler-run")
-	e.initProgress.Modify(pb.WithConstLeft("Run"))
+	e.initProgress.Modify(pb.WithConstLeft(" Run"))
```

Comment:
As I mention above, this looks strange when I run a script - try one with `sleep(1)` in the init phase and `setup()` to see it more slowly

Reply:
Reverted in 20b988aa.